src/renderers/shaders/ShaderChunk/clipping_planes_fragment.glsl

```diff
@@ -0,0 +1,11 @@
+#if NUM_USER_CLIPPING_PLANES > 0
+
+	for ( int i = 0; i < NUM_USER_CLIPPING_PLANES; ++ i ) {
+
+		vec4 plane = userClippingPlanes[ i ];
+		if ( dot( vViewPosition, plane.xyz ) > plane.w ) discard;
```

Comment:
Feel free to add "signedPlaneDistance" to somewhere here:

https://github.com/mrdoob/three.js/blob/master/src/renderers/shaders/ShaderChunk/common.glsl#L58